### DISCUSSION TITLE
Is there a chance for testing datasets and training code to be published?

### DISCUSSION THREAD

**Discussion by mstopa** [snakers4/silero-vad#195] [Q&A] [answered]
Hey Team, first of all thank you for your great work. I found your response from [2021](https://github.com/snakers4/silero-vad/wiki/FAQ#papers-datasets-training-code-etc) saying that testing datasets and training code were not going to get published back then. Do you think that there is a place for a collab within which you could share those resources in their current state (even if not clean and finished)?

**Comment by snakers4** [comment, marked as answer]
Hi,

No such plans yet.
